Src/Newtonsoft.Json.UnityConverters.Tests/Newtonsoft.Json.UnityConverters.Tests.csproj

```diff
@@ -1,10 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net4.7.2</TargetFramework>
+    <TargetFrameworks>net4.7.2;netcoreapp3.0</TargetFrameworks>
```

Comment:
Added TargetFramework here, but could perhaps disable the building of ALL target frameworks in the CircleCI build. We're only interested in the .NET 4.7.2 build in the cloud build anyway.